tests/drivers/pwm/pwm_loopback/src/test_pwm_loopback.c

```diff
@@ -98,9 +100,15 @@ static void test_capture(uint32_t period, uint32_t pulse, enum test_pwm_unit uni
 	}
 
 	if (flags & PWM_CAPTURE_TYPE_PULSE) {
-		zassert_within(pulse_capture, pulse, pulse / 100,
-			       "pulse capture off by more than 1%");
+		if (flags & PWM_POLARITY_INVERTED) {
+			zassert_within(pulse_capture, period - pulse, (period - pulse) / 100,
+					"pulse capture off by more than 1%");
+		} else {
+			zassert_within(pulse_capture, pulse, pulse / 100,
+					"pulse capture off by more than 1%");
+		}
```

Comment:
This change is not really related to missing `pwm_disable_capture()` calls. It should be done in a separate commit.

Reply:
Moved to a separate commit.